azure-pipelines.yml

```diff
@@ -3,10 +3,13 @@ resources:
     - repository: templates
       type: github
       name: NSLS-II/profile-collection-ci
-      ref: refs/heads/collection-2021-1.0
+      ref: refs/heads/main
       endpoint: github
 
 jobs:
-- template: azure-linux.yml@templates  # Template reference
-  parameters:
-    beamline_acronym: ARPES
+  - template: collection-2021-1.0.yml@templates
+    parameters:
+      beamline_acronym: ARPES
+  - template: collection-2021-1.2.yml@templates
+    parameters:
+      beamline_acronym: ARPES
```

Comment:
I think we can use the latest configs for these tests, such as in https://github.com/NSLS-II-TES/profile_simulated_hardware/blob/master/azure-pipelines.yml.